gettsim/benefits/benefit_checks.py

```diff
@@ -14,6 +14,9 @@ def benefit_priority(household, params):
     If the household need cannot be covered via Wohngeld, he has to apply for ALG2.
     There is no way you can receive ALG2 and Wohngeld/Kinderzuschlag at the same time!
     """
+    # Before 2005 no Unemployment benefit is calculated and thus return a nan dataframe.
```

Comment:
Why no unemployment benefit (=ALG 1) ? This should be trivial. Again, raise error if anything in case this concerns ALG 2 (=social assistance).

Reply:
In the docstring above Unemployment Benefit is mapped to ALG2. It is because of ALG 2.

Reply:
That mapping is wrong. ALG 2 is a label that makes no sense in economic terminology.

Reply:
Some better idea @Eric-Sommer ?

Reply:
In general terms, ALG 2 is a *social benefit*.

Reply:
Changed comment and docstring.